applications/ipc_radio/Kconfig

```diff
@@ -0,0 +1,53 @@
+#
+# Copyright (c) 2023 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+mainmenu "Nordic ipc_radio firmware"
+
+config IPC_RADIO_802154
+	bool "IPC IEEE 802.15.4 radio"
+	select NRF_802154_SER_RADIO
+	help
+	  Enable the IPC IEEE 802.15.4 radio serialization.
+
+config IPC_RADIO_BT
+	bool "IPC Bluetooth"
+	help
+	  Enable the IPC Bluetooth radio serialization.
```

Comment:
Shouldn't this configutration depend on SOC_NRF5340_CPUNET?

Reply:
I am not sure. In sample.yaml there are entries that this application can only be run on nrf5340 capable boards.
If I add the "depends on" than no Kconfig will be present and the application will fail with no radio ipc selected. Do you think that's an ok option?